src/IssueViz.Security/ReportView/GroupFileViewModel.cs

```diff
@@ -44,10 +44,11 @@ public void ApplyFilter(ReportViewFilterViewModel reportViewFilter)
     {
         var issueTypesToShow = reportViewFilter.IssueTypeFilters.Where(x => x.IsSelected).Select(x => x.IssueType);
         var filteredIssues = AllIssues.Where(issue => issueTypesToShow.Contains(issue.IssueType));
-        if (reportViewFilter.SelectedSeverityFilter != DisplaySeverity.Any)
-        {
-            filteredIssues = filteredIssues.Where(vm => vm.DisplaySeverity == reportViewFilter.SelectedSeverityFilter);
-        }
+
+        filteredIssues = filteredIssues
+            .Where(vm => DisplaySeverityComparer.Instance.Compare(vm.DisplaySeverity, reportViewFilter.SelectedSeverityFilter) >= 0)
+            .OrderByDescending(vm => vm.DisplaySeverity, DisplaySeverityComparer.Instance);
```

Comment:
Why not something like this, without the comparer?

```suggestion
        filteredIssues = filteredIssues
            .Where(vm => vm.DisplaySeverity >= reportViewFilter.SelectedSeverityFilter)
            .OrderByDescending(vm => vm.DisplaySeverity);
```